mate-user-guide/C/gospanel.xml

```diff
@@ -1267,27 +1267,27 @@ See <xref linkend="panels-addobject"/> for more on this.</para>
     <para>You can add the following types of menu to your panels: </para>
     <itemizedlist>
       <listitem>
-        <para><guimenu>Menu Bar</guimenu>: You can access almost
+        <para><guimenu>Classic Menu</guimenu>: You can access almost
 all the standard applications, commands, and configuration options from
-the menus in the Menu Bar. It contains the <guimenu>Applications</guimenu>, 
+the menus in the Classic Menu. It contains the <guimenu>Applications</guimenu>, 
```

Comment:
`the menus in the Classic Menu` could be shortened down to `the classic menu`

Reply:
I agree that the original wording is poor. Technically accurate, but confusing nevertheless. Not a blocker, though, as only the name is being changed.